src/many-cli/src/main.rs

```diff
@@ -159,10 +182,41 @@ fn main() {
             }
         }
         SubCommand::Message(o) => {
-            // If `pem` is not provided, use anonymous and don't sign.
-            let key = o.pem.map_or_else(CoseKeyIdentity::anonymous, |p| {
-                CoseKeyIdentity::from_pem(&std::fs::read_to_string(&p).unwrap()).unwrap()
-            });
+            let key = if o.module.is_some() && o.slot.is_some() && o.keyid.is_some() {
+                trace!("Getting user PIN");
+                let pin = rpassword::prompt_password("Please enter the HSM user PIN: ").unwrap();
```

Comment:
What are you thoughts on non-interactive TTY?

Reply:
Please test it. We might just want to disallow non-interactive TTY and tell the user (there's an `atty` crate to get caps of your STDIN)

Reply:
Right now, doing `echo 1234 | many message ...` blocks on the password prompt, i.e., it doesn't feed the `echo` as the password.

Reply:
Do you want me to add an `atty` check and terminate the process if it's not a TTY?

Reply:
That's perfect, that's what I would expect. We can error out on it but that's probably fine.